### ISSUE TITLE
Cannot attachMovie() to a movieclip once unloadMovie() has been called on it

### ISSUE THREAD

**Issue by ChrisCPI** [ruffle-rs/ruffle#21683]
### Describe the bug

This test makes an empty MovieClip called "test_mc", and then immediately has `unloadMovie()` called on it. After a 200ms timeout, it then calls `attachMovie()` to attach an empty clip that has AS linkage, and then draws a blue rectangle.

In Ruffle, it is unable to attach the movie, printing this warning:
```
WARN core/src/avm1/globals/movie_clip.rs:814 Unable to attach 'testclip'
```
As such, the blue rectangle never appears.

### Expected behavior

It should successfully attach the clip, and the blue rectangle drawn via `with()` should appear.

### Content Location

[unloadmovie_bug.zip](https://github.com/user-attachments/files/22376583/unloadmovie_bug.zip)

### Affected platform

Desktop app

### Operating system

macOS 15.6.1

### Additional information

This does not happen when using `createEmptyMovieClip()` to create a new clip inside test_mc.

**Comment by danielhjacobs** [comment]
Sounds like this may be the cause of #6254

**Comment by ChrisCPI** [comment]
Interestingly, this also does not work:
```as
function attachMovieToTest()
{
   test_mc.createEmptyMovieClip("test2_mc",test_mc.getNextHighestDepth());
   test_mc.test2_mc.attachMovie("testclip","test3_mc",1);
   with(test_mc.test2_mc.test3_mc)
   {
      beginFill(255);
      moveTo(200,200);
      lineTo(300,200);
      lineTo(300,300);
      lineTo(200,300);
      lineTo(200,200);
      endFill();
   }
}
var test_mc = this.createEmptyMovieClip("test_mc",this.getNextHighestDepth());
test_mc.unloadMovie();
setTimeout(attachMovieToTest,200);
```
`test2_mc` is able to be made inside of `test_mc`, but it fails to attach `test3_mc` to `test2_mc`.
